crates/dips/proto/indexer.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package graphprotocol.indexer.dips;
 
-service DipsService {
+service IndexerDipsService {
```

Comment:
I would suggest naming it `GatewayDipsService.`

```suggestion
service GatewayDipsService {
```
I want to rename the "Dipper" to DIPs Gateway at some point in the future.